client/app/project/project.controller.js

```diff
@@ -936,7 +936,11 @@
          * @returns {string}
          */
         vm.getGpxDownloadURL = function(){
-            return editorService.getGPXUrl(vm.projectData.projectId, vm.getSelectTaskIds(), true);
+            //return editorService.getGPXUrl(vm.projectData.projectId, vm.getSelectTaskIds(), true);
```

Comment:
can we still use the editor service or does the functionality need to be moved here? If it needs to be here, then remove line 939 please